docs/user/azure/install_upi_azurestack.md

```diff
@@ -0,0 +1,591 @@
+# Install: User-Provided Infrastructure on Azure Stack Hub
+
+The steps for performing a user-provided infrastructure install are outlined here. Several
+[Azure Resource Manager][azuretemplates] templates are provided to assist in
+completing these steps or to help model your own. You are also free to create
+the required resources through other methods; the templates are just an
+example.
+
+## Prerequisites
+
+* all prerequisites from [README](README.md)
+* the following binaries installed and in $PATH:
+  * [openshift-install][openshiftinstall]
+    * It is recommended that the OpenShift installer CLI version is the same of the cluster being deployed. The version used in this example is 4.3.0 GA.
+  * [az (Azure CLI)][azurecli] installed and aunthenticated
+    * Commands flags and structure may vary between `az` versions. The recommended version used in this example is 2.0.80.
+  * python3
+  * [jq][jqjson]
+  * [yq][yqyaml] (N.B. there are multiple versions of `yq`, some with different syntaxes)
```

Comment:
IMO, this should be the same version described in the Azure UPI document. If not, be specific about which one is being used in the examples here.

Reply:
+1 I have updated to use the same yq (installed by pip)